test/test-lookup.js

```diff
@@ -49,7 +49,7 @@ test('lookup[getLookupTable]:', function (t) {
   }
   t.ok(table, 'table should exist');
   t.ok(table.lodash, 'lodash should be in the table');
-  t.ok(table.lodash.replace, 'lodash should need to be replaced');
+  t.ok(table.lodash.skip, 'lodash should contain a skip paramter');
```

Comment:
I just changed it from replace to skip as replace was removed
https://github.com/nodejs/citgm/blob/master/lib/lookup.json#L4